frontend/public/components/replication-controller.jsx

```diff
@@ -69,16 +79,79 @@ export const ReplicationControllersDetailsPage = props => <DetailsPage
 
 const kind = 'ReplicationController';
 
+const tableColumnClasses = [
+  classNames('col-lg-2', 'col-md-2', 'col-sm-4', 'col-xs-6'),
+  classNames('col-lg-2', 'col-md-2', 'col-sm-4', 'col-xs-6'),
+  classNames('col-lg-3', 'col-md-4', 'col-sm-4', 'hidden-xs'),
+  classNames('col-lg-1', 'col-md-2', 'hidden-sm', 'hidden-xs'),
+  classNames('col-lg-1', 'col-md-2', 'hidden-sm', 'hidden-xs'),
+  classNames('col-lg-3', 'hidden-md', 'hidden-sm', 'hidden-xs'),
+  Kebab.columnClass,
+];
+
 const ReplicationControllerTableRow = ({obj, index, key, style}) => {
+  const phase = _.get(obj, ['metadata', 'annotations', 'openshift.io/deployment.phase']);
+
   return (
-    <WorkloadTableRow obj={obj} index={index} key={key} style={style} menuActions={replicaSetMenuActions} kind={kind} />
+    <TableRow id={obj.metadata.uid} index={index} trKey={key} style={style}>
+      <TableData className={tableColumnClasses[0]}>
+        <ResourceLink kind={kind} name={obj.metadata.name} namespace={obj.metadata.namespace} title={obj.metadata.uid} />
+      </TableData>
+      <TableData className={classNames(tableColumnClasses[1], 'co-break-word')}>
+        <ResourceLink kind="Namespace" name={obj.metadata.namespace} title={obj.metadata.namespace} />
+      </TableData>
+      <TableData className={tableColumnClasses[2]}>
+        <LabelList kind={kind} labels={obj.metadata.labels} />
+      </TableData>
+      <TableData className={tableColumnClasses[3]}>
+        <Link to={`${resourcePath(kind, obj.metadata.name, obj.metadata.namespace)}/pods`} title="pods">
+          {obj.status.replicas || 0} of {obj.spec.replicas} pods
+        </Link>
+      </TableData>
+      <TableData className={tableColumnClasses[4]}>
+        { phase }
+      </TableData>
+      <TableData className={tableColumnClasses[5]}>
+        <Selector selector={obj.spec.selector} namespace={obj.metadata.namespace} />
+      </TableData>
+      <TableData className={tableColumnClasses[6]}>
+        <ResourceKebab actions={replicaSetMenuActions} kind={kind} resource={obj} />
+      </TableData>
+    </TableRow>
   );
 };
 ReplicationControllerTableRow.displayName = 'ReplicationControllerTableRow';
 
 
 const ReplicationControllerTableHeader = () => {
-  return WorkloadTableHeader();
+  return [
+    {
+      title: 'Name', sortField: 'metadata.name', transforms: [sortable],
+      props: { className: tableColumnClasses[0] },
+    },
+    {
+      title: 'Namespace', sortField: 'metadata.namespace', transforms: [sortable],
+      props: { className: tableColumnClasses[1] },
+    },
+    {
+      title: 'Labels', sortField: 'metadata.labels', transforms: [sortable],
+      props: { className: tableColumnClasses[2] },
+    },
+    {
+      title: 'Status', sortFunc: 'numReplicas', transforms: [sortable],
+      props: { className: tableColumnClasses[3] },
+    },
+    {
+      title: 'Phase', sortFunc: 'metadata.annotations.openshift.io/deployment', transforms: [sortable],
```

Comment:
This `sortFunc` won't work. Try

```suggestion
      title: 'Phase', sortField: 'metadata.annotations["openshift.io/deployment.phase"]', transforms: [sortable],
```